package/MDAnalysis/lib/formats/src/xdrfile.c

```diff
@@ -2510,8 +2510,8 @@ static int xdrstdio_getlong (XDR *, int32_t *);
 static int xdrstdio_putlong (XDR *, int32_t *);
 static int xdrstdio_getbytes (XDR *, char *, unsigned int);
 static int xdrstdio_putbytes (XDR *, char *, unsigned int);
-static off_t xdrstdio_getpos (XDR *);
-static int xdrstdio_setpos (XDR *, off_t, int);
+static int xdrstdio_getpos (XDR *);
+static int xdrstdio_setpos (XDR *, int, int);
```

Comment:
So this change cannot be merged. `off_t` is a large integer by default allowing us to read large files >4GB. Rather use a macro and type def at the top of the file.

Reply:
The unit tests fail when we emulate large files on travis.

Reply:
Just re-iterate @kain88-de : `off_t` has to stay, you need to find a way to support LFS on windows. This would be a deal-breaker.